pull-topic-as-local/pull-topic-as-local

```diff
@@ -32,7 +32,13 @@ for i in $@; do
 	fi
 
 	echo "Pulling topic repository: $i ..."
+	arch=`uname -m`
+	[ $arch == "mips64" ] && arch="loongson3"
+	[ $arch == "aarch64" ] && arch="arm64"
```

Comment:
You might also want to add translation for other architectures, see `arch/` in Autobuild4.